components/kflogin/src/login.js

```diff
@@ -63,8 +63,8 @@ class Login extends Component {
 
   handleClick(){
     try {
-      const hostname = window.location.hostname;
-      axios.post("https://" + hostname + "/apikflogin", {"req-num": "1"}, {
+      const origin = window.location.origin;
```

Comment:
It might be better to replace `window.location.hostname` with `window.location.host`.
You'll still get the port added, but I believe origin will null if you access it without going cross domain.